Header header 

int32 num_lines
geometry_msgs/Point[] start 
geometry_msgs/Point[] end
